### PR TITLE
threadscope 0.2.6 (new formula)

### PR DESCRIPTION
**Pull request by pthariensflame**
This was originally Homebrew/homebrew#37894, but I was told to move it here.

Advice on improving the tests would be greatly appreciated.  :)

### CODE REVIEW

File: threadscope.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  homepage "https://wiki.haskell.org/ThreadScope"
+  url "https://hackage.haskell.org/package/threadscope-0.2.6/threadscope-0.2.6.tar.gz"
+  sha256 "ca3b3a8f57315f47f7f6787e5d92ca26c216fb67562192ae37a8cb37dceecc5f"
+
+  depends_on "pkg-config" => :build
+  depends_on "cabal-install" => :build
+  depends_on "ghc"
+  depends_on "gmp"
+  depends_on :x11
```
Comment (mistydemeo) [inline threadscope.rb]: Minor nitpick, but symbol deps go at the top of the list (above build deps and regular deps). The rest of the deps should also be alphabetized within their categories, too.

**Comment by mistydemeo** [comment]
Does threadscope have any headless functionality, or is `--version` the only thing you can do without spinning up a GUI?

**Comment by pthariensflame** [comment]
I don't think it can be used headlessly; its documentation certainly doesn't make mention of it.  There are a couple of "test" flags you can pass, but they all produce a GUI and then leave it running indefinitely.  :/

File: threadscope.rb (hunk 1 above)
Comment (pthariensflame) [inline threadscope.rb]: I will correct this, then.  :)

**Comment by mistydemeo** [comment]
That's fine then, it doesn't sound like there's much more of a test you can do for this software!

**Comment by pthariensflame** [comment]
@mistydemeo Alterations done!

File: threadscope.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  sha256 "ca3b3a8f57315f47f7f6787e5d92ca26c216fb67562192ae37a8cb37dceecc5f"
+
+  depends_on :x11
+  depends_on "cabal-install" => :build
+  depends_on "pkg-config" => :build
+  depends_on "cairo"
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "gettext"
+  depends_on "ghc"
```
Comment (mistydemeo) [inline threadscope.rb]: Is this needed at runtime, or is it buildtime-only?

Comment (pthariensflame) [inline threadscope.rb]: It's needed at runtime.  ThreadScope is a graphical parallel profiling tool for Haskell, so it needs access to the compiler (and its ABI) to do its job.

**Comment by mistydemeo** [comment]
Great, thanks! Pulled.

**Pull request closed without merging**